extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -40,6 +40,9 @@ class AmpLightbox extends AMP.BaseElement {
 
     /**  @private {?function(this:AmpLightbox, Event)}*/
     this.boundCloseOnEscape_ = null;
+
+    /** @private {boolean} */
+    this.isScrollable_ = false;
```

Comment:
It's going to be expensive to do that since we can't just measure once as mutations like `amp-collapse`  media queries, etc.. will change whether there needs to be scrolling or not.